services/traction/api/endpoints/routes/innkeeper.py

```diff
@@ -103,3 +103,19 @@ async def make_tenant_issuer(
         )
         tenant_issuer = await issuer_repo.create(new_issuer)
     return tenant_issuer
+
+
+@router.delete(
```

Comment:
are we leaving this as permanent? prefer just `DELETE /tenants/{tenant_id}?hard=true` or `DELETE /tenants/{tenant_id}?force=true`

Reply:
i wanted it to be very distinct from a normal operational delete for now, borderline awkward.

if we do it 'correctly' then i think we can probably do something closer to what you have suggested here.